website/prettier.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('prettier').Options} */
+module.exports = {
+  singleQuote: true,
+  semi: false,
```

Comment:
**issue:** Not using semi colons in ts / js files might lead to unexpected behavior, please add semi colons in your code